packages/frontend/src/styles.css

```
@@ -262,6 +262,69 @@ h1 {
+
+.blend-json {
+  border: 1px solid rgba(22, 33, 31, 0.16);
+  border-radius: 8px;
+  background: rgba(255, 255, 255, 0.84);
+  color: #16211f;
+  margin: 0 0 0.75rem;
+  max-height: 240px;
+  overflow: auto;
+  padding: 0.75rem;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)